phases/ephemeral/witx/wasi_ephemeral_profile.witx

```diff
@@ -0,0 +1,27 @@
+(use "wasi_ephemeral_args.witx")
+(use "wasi_ephemeral_clock.witx")
+(use "wasi_ephemeral_environ.witx")
+(use "wasi_ephemeral_fd.witx")
+(use "wasi_ephemeral_path.witx")
+(use "wasi_ephemeral_poll.witx")
+(use "wasi_ephemeral_proc.witx")
+(use "wasi_ephemeral_random.witx")
+(use "wasi_ephemeral_sched.witx")
+(use "wasi_ephemeral_sock.witx")
+
+;;; The WASI standard specifies executables as having all of the
```

Comment:
I was initially confused here about the relationship between "executables" as specified here and [commands](https://github.com/WebAssembly/WASI/blob/master/design/application-abi.md#current-unstable-abi). It looks like an executable is a command that has access to all the path/clock/fd/etc. APIs, is that right? In any case, could you add a comment about this?

Reply:
This is embarrassing, I got "executable" and "command" crossed in my head and named these things wrong. Will go fix them all now.